pkg/pillar/hypervisor/kvm.go

```diff
@@ -791,17 +884,55 @@ func isVncShimVMEnabled(
 	return config.EnableVnc && (config.EnableVncShimVM || globalShimVnc)
 }
 
+func getFmlCustomResolution(status *types.DomainStatus, globalConfig *types.ConfigItemValueMap) (string, error) {
+	fmlResolutions := status.FmlCustomResolution
+	// if not set in the domain status, try to get it from the global config
+	if fmlResolutions == types.FmlResolutionUnset {
+		if globalConfig != nil {
+			item, ok := globalConfig.GlobalSettings[types.FmlCustomResolution]
+			if ok {
+				fmlResolutions = item.StringValue()
+			}
+		}
+	}
+
+	// debug, remove later
+	logError("getFmlCustomResolution -> FmlResolution is set to: %s", fmlResolutions)
```

Comment:
After our meticulous testing, this line can be removed, it served us well.

Reply:
Will do

Reply:
done